server/chart_config.json

```diff
@@ -59,7 +59,7 @@
   },
   {
     "category": "Economics",
-    "titleId": "Unemployment rate: urban vs rural",
+    "titleId": "CHART_TITLE-Unemployment_rate-urban-v-rural",
```

Comment:
wonder if we could have some automatic schemes to update the IDs (and also enforce the ID format from test?)

Reply:
yea, tests would be good. not sure about automatic scheme - really the prefix is all that's required, and a test could catch that (and easy to add a prefix).

i added a basic test to check that the right files were updated in https://github.com/datacommonsorg/website/pull/1078/commits/2801550f6450fb5792f8117375de6b6b8ff74831